modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,274 @@
+
+. Apply the `yaml` by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f tap-example.yaml
+----
+
+. Create a performance profile by following the guidance in "Creating a performance profile".
+
+. Create the following `Pod` spec, and then save the YAML in the `dpdk-pod-rootless.yaml` file.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dpdk-app
+  namespace: test-namespace <1>
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+      {"name": "sriov-network", "namespace": "test-namespace"},
+      {"name": "tap-one", "interface": "ext0", "namespace": "test-namespace"}]'
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: "worker-0"
+  securityContext:
+      fsGroup: 1001 <2>
+      runAsGroup: 1001 <3>
+      seccompProfile:
+        type: RuntimeDefault
+  containers:
+  - name: testpmd
+    image: <DPDK_image> <4>
+    securityContext:
+      capabilities:
+        drop: ["ALL"] <5>
+        add: <6>
+          - IPC_LOCK
+          - NET_RAW #for mlx only <7>
+      runAsUser: 1001 <8>
+      privileged: false <9>
+      allowPrivilegeEscalation: true <10>
+      runAsNonRoot: true <11>
+    volumeMounts:
+    - mountPath: /mnt/huge <12>
+      name: hugepages
+    resources:
+      limits:
+        openshift.io/sriovnic: "1" <13>
+        memory: "1Gi"
+        cpu: "4" <14>
+        hugepages-1Gi: "4Gi" <15>
+      requests:
+        openshift.io/sriovnic: "1"
+        memory: "1Gi"
+        cpu: "4"
+        hugepages-1Gi: "4Gi"
+    command: ["sleep", "infinity"]
+  runtimeClassName: performance-cnf-performanceprofile <16>
+  volumes:
+  - name: hugepages
+    emptyDir:
+      medium: HugePages
+----
++
+--
+<1> Specify the same `target_namespace` where the `SriovNetwork` object is created. If you want to create the pod in a different namespace, change `target_namespace` in both the `Pod` spec and the `SriovNetwork` object.
+<2> Sets the group ownership of volume-mounted directories and files created in those volumes.
+<3> Controls which primary group ID the containers are run with.
+<4> Specify the DPDK image which includes your application and the DPDK library used by application.
+<5> Dropping all capabilities (`ALL`) from the container's securityContext means that the container will not have any special privileges beyond what is necessary for normal operation.
+<6> Specify additional capabilities required by the application inside the container for hugepage allocation, system resource allocation, and network interface access the Virtual Function I/O (VFIO) device.
+<7> Mellanox network interface controller (NIC) requires the `NET_RAW` capability.
+<8> Controls which user ID the container is run with.
+<9> This setting indicates that the container(s) within the pod should not be granted privileged access to the host system.
+<10>  This setting allows a container to escalate its privileges beyond the initial non-root privileges it may have been assigned.
```

Comment:
```suggestion
<10>  This setting allows a container to escalate its privileges beyond the initial non-root privileges it might have been assigned.
```